core/src/main/java/org/jdbi/v3/core/generic/internal/Preconditions.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2007 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.jdbi.v3.core.generic.internal;
+
+import javax.annotation.Nullable;
+
+public final class Preconditions {
+  private Preconditions() {}
+
+  public static void checkArgument(boolean expression) {
```

Comment:
Do we actually use this zero-argument variety?  Seems like we should always have a message and that using this is almost certainly wrong.